src/Hazelcast.Net/DistributedObjects/IHVectorCollection.cs

```diff
@@ -18,6 +18,7 @@
 namespace Hazelcast.DistributedObjects
 {
     /// <summary>
+    /// **BETA**
     /// Represents a distributed collection of vector documents.
```

Comment:
Do you also need to put the BETA on `IVectorDocument`?

Reply:
Sure.